PWA/dist/index.html

```diff
@@ -1 +1 @@
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><!--[if IE]><link rel="icon" href="/favicon.ico"><![endif]--><link href=https://cdn.jsdelivr.net/npm/bootstrap@4.0.0-beta/dist/css/bootstrap.min.css rel=stylesheet><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel=stylesheet><title>Vue-PWA-Rapid-Prototype</title><script src=https://unpkg.com/vue></script><script src=https://unpkg.com/vue-upload-component></script><link href=/css/app.14e4f460.css rel=preload as=style><link href=/css/chunk-vendors.7caec713.css rel=preload as=style><link href=/js/app.58f89673.js rel=preload as=script><link href=/js/chunk-vendors.6acf834f.js rel=preload as=script><link href=/css/chunk-vendors.7caec713.css rel=stylesheet><link href=/css/app.14e4f460.css rel=stylesheet><link rel=icon type=image/png sizes=32x32 href=/img/icons/favicon-32x32.ico><link rel=icon type=image/png sizes=16x16 href=/img/icons/favicon-32x32.ico><link rel=manifest href=/manifest.json><meta name=theme-color content=#4DBA87><meta name=apple-mobile-web-app-capable content=no><meta name=apple-mobile-web-app-status-bar-style content=default><meta name=apple-mobile-web-app-title content="Vue PWA Rapid Prototype"><link rel=apple-touch-icon href=/img/icons/apple-touch-icon-152x152.png><link rel=mask-icon href=/img/icons/safari-pinned-tab.svg color=#4DBA87><meta name=msapplication-TileImage content=/img/icons/msapplication-icon-144x144.png><meta name=msapplication-TileColor content=#000000></head><body><noscript><strong>We're sorry but Vue-PWA-Rapid-Prototype doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.6acf834f.js></script><script src=/js/app.58f89673.js></script></body></html>
```

Comment:
This directory is the production-ready optimized code, it should be built via an npm command after the repository is cloned, therefore it should not be uploaded to github.